python/paddle/fluid/framework.py

```diff
@@ -523,10 +524,15 @@ def find_name(var_list, name):
                             % (in_proto.name, len(in_args)))
                     in_arg_names = []
                     for arg in in_args:
-                        if isinstance(arg, basestring):
+                        if issubclass(arg.__class__, six.string_types):
```

Comment:
can you explain this block of change?

Reply:
```basestring``` is no longer available in Python 3. from Python3's offical release: ```The builtin ```basestring``` abstract type was removed. Use ```str``` instead. ```

Reply:
why is it changed to issubclass?